server/broadcast.go

```diff
@@ -285,6 +283,9 @@ func transcodeSegment(cxn *rtmpConnection, seg *stream.HLSSegment, name string)
 			monitor.SegmentTranscodeFailed(monitor.SegmentTranscodeErrorNoOrchestrators, nonce, seg.SeqNo, errNoOrchs, true)
 		}
 		glog.Infof("No sessions available for segment nonce=%d seqNo=%d", nonce, seg.SeqNo)
+		// We may want to introduce a "non-retryable" error type here
+		// would help error propagation for live ingest.
+		// similar to the orchestrator's RemoteTranscoderFatalError
```

Comment:
perhaps move to using `github.com/cenkalti/backoff`  ....   this can be used to handle retries and retry limits and time between retries, etc

Reply:
We don't really want to back off here (retry immediately with another O), although whether we should stop after a certain number of retries is something that can be discussed separately.

Reply:
I'm suggesting the fact that the cenktali package includes a solution for non-retryable errors.  Especially if we already have that package linked to this project, it makes sense to use its non-retryable error flag.